Flabbergasted-Flamingos/Bangazon-admin-app
# changed first test

# Description
Add description about what this pull request covers.

## Number of Fixes
Type of number of fixes included in this PR.

## Related Ticket(s)
List related ticket(s) that apply to the work done in this pull request

## Problem to Solve
Changed first test to test if the response has a department collection in the context.

## Proposed Changes
Describe the proposed changes.

## Expected Behavior
Describe the expected behavior.

## Steps to Test Solution
```
git fetch --all
git checkout cd_fix_departments_test
python manage.py test
```
Both tests should pass

## Testing

[ ] There are new unit tests in this PR, and I verify that there is full coverage of all new code.
[ ] I certify that all existing tests pass

## Documentation
[ ] There is new documentation in this pull request that must be reviewed..
[ ] I added documentation for any new classes/methods

## Deploy Notes
Notes regarding deployment the contained body of work.  These should note any
db migrations, etc.
